shared-bindings/bleio/Central.c

```diff
@@ -139,6 +130,47 @@ STATIC mp_obj_t bleio_central_disconnect(mp_obj_t self_in) {
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_1(bleio_central_disconnect_obj, bleio_central_disconnect);
 
+//|   .. method:: discover_remote_services(service_uuids_whitelist=None)
+//|      Do BLE discovery for all services or for the given service UUIDS,
+//|      to find their handles and characteristics.
+//|      The attribute `remote_services` will contain a list of all discovered services.
```

Comment:
Why not just return the discovered services? Is it async? Returning them would allow the user to decide how long to keep the objects around.

Reply:
I thought I would do this, and then forgot :)